src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingOverrideCheck.java

```diff
@@ -81,6 +108,24 @@
  *       value="true"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Test {
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     public void equals() { // violation, not be annotated with &#64;Override
+ *
+ *     }
+ * }
+ *
+ * class Test extends SuperClass {
```

Comment:
`This property will only check classes, interfaces, etc. that do not contain the extends or implements keyword or are not anonymous classes.`

Please add example for interface and anonymous class

Reply:
ok , I add these.